README.md

```diff
@@ -24,6 +24,42 @@ SolidusSupport::Migration[5.0] # same as `ActiveRecord::Migration[5.0]`
 
 There's no reason to use `SolidusSupport::Migration[5.0]` over `ActiveRecord::Migration[5.0]`, but it is provided.
 
+### Testing Helpers
+
+This gem provides some useful helpers for RSpec to setup an extension's test
+environment easily.
+
+Into your extension's spec/spec_helper.rb:
+
+```ruby
+require "solidus_support/extension/feature_helper"
+```
+
+This helper loads configuration needed to run extensions feature specs
+correctly, setting up Capybara and configuring Rails test application
+to precompile assets before the first feature spec.
+
+To run, this helper needs to require a `rails_helper`, also provided by
+this gem and requireable as a stand-alone helper.
+
+By doing:
+
+```ruby
+require "solidus_support/extension/rails_helper"
+```
+
+extension's test suite will have all Rails related tests configuration,
+like authorization helpers, Solidus core factories, url helpers, and
+other helpers to easily work with Solidus Config.
+
+This `rails_helper` in turn requires a basic `spec_helper`, which is
```

Comment:
> This `rails_helper` in ~turn requires a~ basic `spec_helper`, which is

This `rails_helper` in return requires the basic `spec_helper`, which is

Reply:
I know I can be wrong here but I think `in turn` here is correct. 😄 🇬🇧

Reply:
I am also not sure. Leave it as it is then :)